src/quantum/HISTORY.rst

```diff
@@ -6,4 +6,10 @@ Release History
 0.11.2906.2
 ++++++
 * Initial release. Version intended to work with Azure Quantum Private Preview 
-  and with QDK version 0.11.2906.*
\ No newline at end of file
+  and with QDK version 0.11.2906.*
+
+0.13.2011.1901
```

Comment:
Changelog history should be in descending order, with newest changest at the top.